documentation/4.0/content/managing-domains/accessing-the-domain/istio/istio.md

```diff
@@ -37,7 +37,7 @@ The current support for Istio has these limitations:
   and has been tested with single and multicluster
   Istio installations from 1.10 up to 1.13.2.
 
-* For using Istio in Red Hat OpenShift, this operator version 4.0 is only compatible with Red Hat OpenShift version 4.11.x with the latest Service Mesh 2.2.3 and above.  
+* When using Istio in Red Hat OpenShift, operator version 4.0 and later is only compatible with Red Hat OpenShift version 4.11.x with the latest Service Mesh 2.2.3 and later.  
```

Comment:
only compatible -> compatible only 
(Here's the guidance: Position of only: Keep only close to that which it modifies to improve the clarity and accuracy of the sentence. For example, don’t present …you may only be interested in verifying that your connector’s endpoints… Instead, put only next to what it’s actually modifying, and in addition avoid splitting the verb, as in …you may be interested only in verifying that your connector’s endpoints…)

Reply:
Thanks @rosemarymarano